java/viewshed-camera/README.md

```diff
@@ -1,24 +1,37 @@
-# Viewshed Camera
-A viewshed shows the visible and obstructed areas from an observer's vantage point. This sample demonstrates how to create and update a viewshed from a camera.
+# Viewshed camera
 
-![Viewshed Camera App](viewshed-camera.png)
+Analyze the viewshed for a camera. A viewshed shows the visible and obstructed areas from an observer's vantage point. 
 
-# How to use the sample
-The sample will start with a viewshed created from the initial camera location, so only the visible (green) portion of the viewshed will be visible. Move around the scene to see the obstructed (red) portion. Click the 'Update from Camera' button to update the viewshed to the current camera position.
+![Image of viewshed camera](viewshed-camera.png)
 
-# How it works
-1. Get a `Camera` either by creating it, or by getting the current camera from the scene with `sceneView.getCurrentViewpointCamera()`.
-2. Create a `LocationViewshed` passing in the `Camera` plus a min/max distance.
-3. To update the viewshed with a new camera, use `viewshed.updateFromCamera(camera)`
+## Use case
+
+A viewshed analysis is a type of visual analysis you can perform on a scene. The viewshed aims to answer the question 'What can I see from a given location?'. The output is an overlay with two different colors - one representing the visible areas (green) and the other representing the obstructed areas (red).
+
+## How to use the sample
+
+The sample will start with a viewshed created from the initial camera location, so only the visible (green) portion of the viewshed will be visible. Move around the scene to see the obstructed (red) portions. Use the 'Update from Camera' button to update the viewshed to the current camera position.
+
+## How it works
+
+1. Get the current camera from the scene with `SceneView.getCurrentViewpointCamera()`.
+2. Create a `LocationViewshed`, passing in the `Camera` and a min/max distance.
+3. Update the viewshed from a camera with `viewshed.updateFromCamera(camera)`.
+
+## Relevant API
 
-# Relevant API
 * AnalysisOverlay
-* ArcGISTiledElevationSource
 * ArcGISScene
```

Comment:
I wonder why it is necessary to list ArcGISScene in the relevant API? I see that in the relatively more complicated Viewshed GeoElement ArcGISScene doesn't merit a mention, but also in the preceding Point Cloud sample it is glossed over as well, with only the PointCloudLayer listed for API.

Reply:
you're right, that should come out along with SceneView. They aren't the main act of this sample. I've removed them, thanks!